integration/secret/secret_test.go

```diff
@@ -41,3 +43,110 @@ func TestSecretInspect(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, secret.ID, secretResp.ID)
 }
+
+func TestSecretList(t *testing.T) {
+	skip.If(t, !testEnv.IsLocalDaemon())
```

Comment:
This is new, why is it necessary to be a local daemon?

Reply:
@dnephin Thanks. It should be checking `OSType != linux`. The PR has been updated.